templates/CRM/common/snippet.tpl

```diff
@@ -34,7 +34,7 @@
           {include file="CRM/common/status.tpl"}
         {/if}
 
-        {if !empty($isForm)}
+        {if !$isForm}
```

Comment:
This seems backwards.
It kind of scares me that it passes tests.

Reply:
@demeritcowboy you are totally right - it was wrong - fixed now